pkg/cmd/util/env.go

```diff
@@ -41,12 +41,19 @@ func GetEnv(key string) (string, bool) {
 
 type Environment map[string]string
 
-var argumentEnvironment = regexp.MustCompile("(?ms)^([\\w\\-_]+)\\=(.*)$")
+var validEnvArgRegex = "(?ms)^([\\w_]+)\\=(.*)$"
```

Comment:
@fabianofranz 
> Also \w already covers underscore.

Thanks! Will fix

> Right, - is not accepted. It's just letters, digits and underscore, so just \w covers it (no need for _). http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap08.html

Ah, `-` always seemed to fail when checked at the server side, not sure why it was in this regex before though.